.github/ISSUE_TEMPLATE/propose-process-change.yml

```diff
@@ -0,0 +1,14 @@
+name: Do you want to propose a change to the EIP process itself?
+description: Use this if you want to propose changes to the EIP process
+title: Title
```

Comment:
Does omitting `title` not work, or is this left over from before the other PR?

Reply:
This is left over. Let me fix it.